com.unity.ml-agents/Runtime/SideChannels/FloatPropertiesChannel.cs

```diff
@@ -32,21 +32,29 @@ public FloatPropertiesChannel(Guid channelId = default(Guid))
         }
 
         /// <inheritdoc/>
-        public override void OnMessageReceived(byte[] data)
+        public override void OnMessageReceived(IncomingMessage msg)
         {
-            var kv = DeserializeMessage(data);
-            m_FloatProperties[kv.Key] = kv.Value;
-            if (m_RegisteredActions.ContainsKey(kv.Key))
+            var key = msg.ReadString();
+            var value = msg.ReadFloat32();
+
+            m_FloatProperties[key] = value;
+            if (m_RegisteredActions.ContainsKey(key))
```

Comment:
`TryGetValue`?

Reply:
Converted all the `ContainsKey`s to `TryGetValue`s